layouts/shortcodes/home-section-resources.html

```diff
@@ -14,36 +14,36 @@
 <section class="container">
   <h2>Resources</h2>
   <div class="row">
-    <div class="col-md-6 col-sm-12 flex-column">
-      <div class="flex-grow text-center match-height-item background-white padding-20 margin-bottom-30">
-        <p><img class="img img-responsive margin-auto padding-30" style="height: 150px; width: 150px;" src="images/resources/whitepaper-icon.svg" alt="resource image"></p>
+    <div class="col-md-6 col-sm-12">
+      <div class="flex-column text-center match-height-item background-white padding-20 margin-bottom-30">
+        <p><img class="img img-responsive margin-auto padding-20" style="height: 150px; width: 150px;" src="images/resources/whitepaper-icon.svg" alt="resource image"></p>
         <h3>Whitepaper</h3>
         <p>Learn how open source is fueling digital transformation in Europe and beyond.</p>
-        <p><a class="btn btn-secondary" href="https://outreach.eclipse.foundation/enabling-digital-transformation-white-paper">Download</a></p>
+        <p class="flex-grow" style="display: flex; align-items: flex-end; justify-content: center;"><a class="btn btn-secondary" href="https://outreach.eclipse.foundation/enabling-digital-transformation-white-paper">Download</a></p>
```

Comment:
Adding these styles to make it work... I wonder if there is better way ? Or can I add these customized styles into style.less? 🤔 I have tried earlier but seems not working...

Reply:
The flex-grow class should be on the text section above the button, not the button paragraph tag. The idea with flex-grow is we cause it to grow downwards so that we fill the container, not grow up to take additional space (while this is possible, its a more complicated way of accomplishing the same thing). Making this change, you don't need any of the styles you added inline.

A good rule of thumb, we want as little as possible inline styles. We want clean, reusable CSS classes wherever possible. This way if we ever convert a piece of functionality to be a component, we don't need to dig through styles to figure out what we need. It also helps keep things consistent.

Reply:
This works great! I did not notice I was confused and added it on the wrong <p> to add flex-grow, I don't like adding inline css either. Thanks for the comments!